model/godleyTableWindow.cc

```diff
@@ -201,7 +201,14 @@ namespace minsky
           {
             CairoSave cs(surface->cairo());
             cairo_move_to(surface->cairo(), x, columnButtonsOffset);
-            colWidgets[col].draw(surface->cairo());
+            // Prevent the existence of more than one equity column in single equity column mode
+            if (godleyIcon->table._assetClass(col)==GodleyAssetClass::equity && col!=godleyIcon->table.cols()-1 && !cminsky().multipleEquities) {
+               godleyIcon->table.deleteCol(col);
```

Comment:
I don't think it is a good idea to abruptly delete columns. Rather, I think that the presence of more than one equity column should be treated identically to the case where multipleEquities is true (ie the old way).

Reply:
Okay, I understand now. I should deal with the number of columns at column draw time and not button widget draw time. Thanks :)